strongbox-web-core/src/test/resources/etc/conf/strongbox.xml

```diff
@@ -0,0 +1,387 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration >
+
+    <instance-name>strongbox</instance-name>
+    <version>${project.version}</version>
+    <revision>${strongbox.revision}</revision>
+
+    <baseUrl>http://localhost:48080/</baseUrl>
+    <port>48080</port>
+
+    <session-configuration timeout-seconds="3600" />
+
+    <remote-repositories-configuration check-interval-seconds="60" heartbeat-threads-number="5">
+        <retry-artifact-download-configuration timeout-seconds="60"
+                                               max-number-of-attempts="5"
+                                               min-attempts-interval-seconds="5"/>
+    </remote-repositories-configuration>
+
+    <cors-configuration allowed-credentials="true" max-age="600">
```

Comment:
I have the feeling that this configuration should be extracted to one of the other `XML` configuration files. Perhaps we should add it as a follow-up task...?